### ISSUE TITLE
Updated HABmin2 jar and now it shows up twice in Paper UI and results in Jersey server errors

### ISSUE THREAD

**Issue by raerae1616** [openhab/org.openhab.ui.habmin#221]
I just tried updating my HABmin jar to the latest snapshot version 2 in Openhab 2, and now I'm getting Jerser web server errors, and I have 2 HABmin links in the Paper UI start screen (or the start screen where you can run Paper UI or HABmin, not sure what that screen is called).

In addition, neither Paper UI or HABmin now work because they result in Jersey server errors after this.

All I did was:
1. Shutdown OpenHAB2
2. delete HABmin 0.1.6 jar
3. Copy over HABmin 2.0.0 Snapshot jar (from Cloudbees)
4. Restart Raspberry Pi and verify Openhab2 services were restarted.

**Any ideas on how to correctly uninstall/update HABmin2 on OH2?**  

I've not been able to find any information in the docs/wiki/google.  Thanks!

**Comment by cdjackson** [comment]
KAraf has probably cached a Copy of the running jar. This  is normal and if your not using the standard bundle install you should check the karat console to see what bundles are running.

**Comment by raerae1616** [comment]
Thanks Chris -- that put me on the right track.  I know verry little about the Karaf environment/container that OpenHAB2 uses.  

For any other newbies the solution was that Karaf caches the jar files internally (somehow, because I scanned the system for the jar filename and couldn't locate it).  But the bundle:list and bundle:uninstall commands helped to clean it up.  Then copying over the latest jar into the addons folder allowed it to be re-installed without conflicts.

So to summarize to update HABmin, this is what I did to get it working:

1. Remove the jar from the addons folder
2. SSH into Karaf and run "bundle:list -s org.openhab.binding.*" to see if the binding is installed
3. If so then make note of the ID from the list and run "bundle:uninstall 222" where 222 was the ID of my HABmin bundle from the list.
4. Copy over the latest *.jar file into the addons folder
5. Restart OpenHAB and it will reinstall the new version.

Hope this helps anyone else that's new to Karaf and OpenHAB!
